Bottom: eaa8b9d02cd1465be862b8e5ca8d53023e7c6115
Top:    c84945fb420a14ea955c36613e32172662c4d474
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:43:59 +0000

Allow running of action scripts when running PCB as an exporter


---

diff --git a/src/main.c b/src/main.c
index e5d93eb..992801c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -991,12 +991,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1029,6 +1023,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString, 0);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
